api/seeds/shared/roles-and-activities.js

```diff
@@ -32,16 +32,7 @@ const roleToActivityMappings = {
     'view-roles',
     'create-roles',
     'edit-roles',
-    'submit-federal-response',
-    'submit-clearance',
-    'edit-comments',
-    'submit-document',
-    'submit-state-response',
-    'create-draft',
-    'edit-document',
-    'edit-response',
-    'delete-users',
-    'view-document'
+    'delete-users'
```

Comment:
[NIST 800-53 AC-6(2)](https://nvd.nist.gov/800-53/Rev4/control/AC-6) says admins shouldn't have access to non-admin functions, so this is a useful change for ATO.